cmd/yorkie/version.go

```diff
@@ -19,26 +19,43 @@ package main
 import (
 	"fmt"
 	"runtime"
+	"runtime/debug"
 
 	"github.com/spf13/cobra"
 
 	"github.com/yorkie-team/yorkie/internal/version"
 )
 
+const commitRevisionKey = "vcs.revision"
+const buildTimeKey = "vcs.time"
+
 func newVersionCmd() *cobra.Command {
 	return &cobra.Command{
 		Use:   "version",
 		Short: "Print the version number of Yorkie",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			fmt.Printf("Yorkie: %s\n", version.Version)
-			fmt.Printf("Commit: %s\n", version.GitCommit)
+			fmt.Printf("Commit: %s\n", getBuildInfoByKey(commitRevisionKey))
 			fmt.Printf("Go: %s\n", runtime.Version())
-			fmt.Printf("Build date: %s\n", version.BuildDate)
+			fmt.Printf("Build date: %s\n", getBuildInfoByKey(buildTimeKey))
 			return nil
 		},
 	}
 }
 
+func getBuildInfoByKey(key string) string {
```

Comment:
It's like a function that returns a value from `BuildInfo.Settings` by key. So it seems more accurate to name it `getSettingValueByKey`. But I think `settingValueByKey` is appropriate because Go doesn't name getters like `GetXXX`.

https://go.dev/doc/effective_go#Getters